src/app/components/BibPage/ElectronicResources.jsx

```diff
@@ -0,0 +1,17 @@
+import { Card, CardContent, CardHeading } from '@nypl/design-system-react-components'
+import generateERLinks from '../../utils/electronicResources'
+import React from 'react';
+
+const ElectronicResources = ({ electronicResources }) => {
```

Comment:
Sorry, I missed this the first time around. Add a comment above this to explain what it does.

```
/**
 * ElectronicResources renders ....
 * @param {array} electronicResources - an array of ...
 */
const ElectronicResources = ({ electronicResources }) => {
```